src/common/Page.ts

```diff
@@ -283,7 +283,7 @@ export const enum PageEmittedEvents {
    * Emitted when a page issues a request and contains a {@link HTTPRequest}.
    *
    * @remarks
-   * The object is readonly. See {@Page.setRequestInterception} for intercepting
+   * The object is readonly. See {@Page setRequestInterception} for intercepting
```

Comment:
We still need the dot here - this should be `@Page.setRequestInterception` so TSDoc links to the right method